tensorflow/core/grappler/utils/topological_sort.cc

```diff
@@ -81,8 +81,8 @@ Status ComputeTopologicalOrder(
     int ready_node = (*ready_nodes)[front];
     for (int fanout : graph_view.GetFanout(ready_node)) {
       ++num_ready_inputs[fanout];
-      if (num_ready_inputs[fanout] ==
-          static_cast<int>(graph_view.GetFanin(fanout).size())) {
+      const int graph_view_GetFanin_fanout_size = graph_view.GetFanin(fanout).size();
+      if (num_ready_inputs[fanout] == graph_view_GetFanin_fanout_size) {
```

Comment:
Local variable name should be `snake_case`, please remove capitalization in `GetFanin` substring